pkg/vm/convertor.go

```diff
@@ -0,0 +1,72 @@
+package vm
+
+import (
+	"math/big"
+
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/vm/stackitem"
+)
+
+// ToContractParameter converts Item to smartcontract.Parameter
+func ToContractParameter(i stackitem.Item, seen map[stackitem.Item]bool) smartcontract.Parameter {
```

Comment:
> Would it be better to put it into smartcontract?

Worth trying, it's somewhere in between `smartcontract` and `stackitem`, certainly not `vm` now. If we can't easily put it into `stackitem`, then we should try putting it somewhere in the `smartcontract`, as this interface user will need both packages anyway.

Reply:
Moved this function to `smartcontract` package and renamed to `ParameterFromStackItem`